text/0000-nft-standard.md

```diff
@@ -0,0 +1,133 @@
+
+## Metadata
+
+Most NFTs also need extra data besides just the digital asset URL. The metadata standard is important to be followed in order to help future platform integrations with Hathor NFTs.
+
+If the NFT requires a metadata, the output script data saved on blockchain **MUST** be the metadata URL in an immutable protocol (e.g. ipfs as explained in the last section), e.g. ipfs://ipfs/<hash>/metadata.json.
+
+The metadata should have the following JSON structure:
+
+```
+{
+    "name": {
+        "type": "string",
+        "description": "Identifies the asset to which this token represents"
+    },
+    "description": {
+        "type": "string",
+        "description": "Describes the asset to which this token represents"
+    },
+    "file": {
+        "type": "string",
+        "description": "A URL pointing to an immutable resource with a digital asset to which this token represents"
+    },
+    "attributes": {
+        "type": "Array<AttributeObject>",
+        "description": "These are the extra attributes for the digital asset. It's an array of AttributeObject to make it as flexible as possible."
+    },
+}
+```
+
+The AttributeObject has the following JSON structure:
+
+```
+{
+    "type": {
+        "type": "string",
+        "description": "Type of the attribute, e.g. rarity, stamina, eyes"
+    },
+    "value": {
+        "type": "string | decimal",
+        "description": "Value of the attribute, e.g. rare, 1.4, blue"
+    }
+}
+```
+
+The AttributeObject may have more attributes than the ones described above but those are the required ones.
+
+### Metadata example
+
+```
+{
+    "name": "Gandalf",
+    "description": "A wizard, one of the Istari order, and the leader and mentor of the Fellowship of the Ring",
+    "file": "ipfs://ipfs/QmbuthvFV2EjvfmWXxt2L83PwPPwbjjggBhVsrEB7AXW123/gandalf.png",
+    "attributes": [
+        {
+            "type": "rarity",
+            "value": "super rare"
+        },
+        {
+            "type": "hp",
+            "value": 32
+        },
+        {
+            "type": "intelligence",
+            "value": 99
+        },
+        {
+            "type": "ring",
+            "value": 0
+        }
+    ]
+}
+```
+
+## Custom NFT
+
+There are some special cases where the NFT token won't follow the proposed standard, e.g. if it needs more than one data output. In that case, our wallets and explorer won't automatically identify this token as NFT.
+
+Given that this situation is expected to be rare, we will handle them manually. The NFT creator will need to get in touch with Hathor team on Discord, in order to have the token identified as an NFT on the official Hathor explorer. Besides that, if the digital asset URL follows the immutable requirements, it can be shown in the explorer just like any other standard NFT.
```

Comment:
Suggestion: Besides that, as long as the digital asset's URL is immutable, it should be approved in the review and should be shown in Hathor's Public Explorer like any other standard NFT.